js/chart/chart-view.js

```diff
@@ -262,7 +255,7 @@ var ChartView = {
     // Draw sentiment
     if (!self.data.sentimentError) {
       // SentimentChart.setProperties();
-      SentimentChart.update();
+      ChartView.update(SentimentChart);
```

Comment:
Because I moved the `update()` method into ChartView as it does the exact same thing in all 3 charts